website/static/js/profile.js

```diff
@@ -539,9 +581,21 @@ var SocialViewModel = function(urls, modes) {
             {label: 'Google Scholar', text: self.scholar(), value: self.scholar.url()}
         ];
     });
-
+        
+    self.profileWebsiteEmpty = ko.computed(function() {
```

Comment:
If the code is refactored to only allow non-empty, valid values in `profileWebsites` as suggested in my previous comment, this computed is unnecessary.